apis/v1beta1/httproute_types.go

```diff
@@ -263,6 +263,56 @@ type HTTPRouteRule struct {
 	// +optional
 	// +kubebuilder:validation:MaxItems=16
 	BackendRefs []HTTPBackendRef `json:"backendRefs,omitempty"`
+
+	// Timeouts defines the timeouts that can be configured for an HTTP request.
+	//
+	// Support: Extended
+	//
+	// +optional
+	// <gateway:experimental>
+	Timeouts *HTTPRouteTimeouts `json:"timeouts,omitempty"`
+}
+
+// HTTPRouteTimeouts defines timeouts that can be configured for an HTTPRoute.
+// Timeout values are formatted like 1h/1m/1s/1ms, as specified in GEP-2257,
+// and MUST BE >= 1ms or "0s" to disable (no timeout).
```

Comment:
To avoid being repetitive and risk this text getting out of sync I think it's best to leave the format description to the definition of that type. 
```suggestion
// Timeout values are represented with Gateway API Duration formatting.
// Specifying a zero value such as "0s" is interpreted as no timeout.
```

Reply:
addressed in ebfc098431820b4fb78d997c655ccfff7df23c62